### ISSUE TITLE
Potential bug in _makeFullCollection

### ISSUE THREAD

**Issue by ileto** [backbone-paginator/backbone-pageable#55]
var fullCollection = new (Backbone.Collection.extend(proto))(models, options);

```
  for (i = 0, length = properties.length; i < length; i++) {
    prop = properties[i];
    if (this[prop] !== thisProto[prop]) {
      fullCollection[prop] = prop; // WHY set the literal prop ?
    }
  }
```

**Comment by wyuenho** [comment]
Haha good catch. It should be `this[prop]`. Care to make a PR and a test case?

**Comment by ileto** [comment]
Hmm, don't know how to do the PR or test case. You do the "hard" work... :+1:

**Comment by wyuenho** [comment]
Fixed in 91869a13f995c4ffe223bfd6d22c8252a31cbaec
